### ISSUE TITLE
Search function and formatting

### ISSUE THREAD

**Issue by sbyt32** [sbyt32/Preordain#23]
Minor worry about how to structure the following without being open to a SQL injection. 
```python
@search_router.get('/{query}')
async def search_for_card(query: str):
    conn, cur = connect_db()
    cur.execute("""
        SELECT * 
        FROM card_info.info 
        WHERE LOWER(card_info.info.name) LIKE LOWER('%Thalia%');
    """)
```

What is the best approach here?

**Comment by sbyt32** [comment]
Is this safe? I found it on https://stackoverflow.com/questions/37273237/
It probably is but something about it feels like it isn't.
```python
@search_router.get('/{query}')
async def search_for_card(query: str):
    conn, cur = connect_db()
    cur.execute("""
    SELECT * FROM card_info.info WHERE lower(card_info.info.name) ILIKE %s 

    """, ('%{}%'.format(query),))
    data = cur.fetchall()
    conn.close()
    if data:
        return data
```

**Comment by sbyt32** [comment]
Merged in ed86d77

Note: 
> That’s safe-ish. Can’t run arbitrary queries, but people can add their own wildcards if they want, e.g. “a%b” in the query. Up to you if that’s acceptable or not.
